### PR TITLE
Update AdministratorUpdatesEnabled with new value

### PR DESCRIPTION
**Pull request by edwardskrod**
##What
Update the AdministratorUpdatesEnabled policy with a new value.

##Why
Admins will need to set AdministratorUpdatesEnabled  to 2 in order to opt client machines into the Automatic Update experience.  This new setting needs to be reflected in the ADMX Template.  

##How
Add an enum element with the various policy setting values.

## Screen Shots

AdministratorUpdates

dropDown

AdminUpdatesEnabled1

AdminUpdatesEnabled2

### CODE REVIEW

**Comment by realDuang** [comment]
邮件已经被成功接收，谢谢您的来信，我将尽快查收。———————————————————————————这是一封神奇的回复信息。得到我的回复信息，有可能会获得好运哟~

**Comment by edwardskrod** [comment]
The PR pipeline is currently failing.

However, I confirmed we have a good build by running the PR against my local fork pipeline.

ForkedPipe

**Comment by skylarnam** [comment]
Quick question: what does presentation do here? 
Also, for the elements, are you taking the values $(string.AdministratorUpdatesEnabled_EnabledForCatalogAndWSUS)" and $(string.AdministratorUpdatesEnabled_EnabledForCatalogWSUSAndAU)"> and using it for your policy as well? For the elements change I tried to look into the "Why" section but couldn't decipher much.

**Review by tydunkel — APPROVED** [review]

File: settingFiles/admx/en-US/VisualStudio.adml [hunk 1]
```diff
@@ -73,6 +73,10 @@ For more information, see https://aka.ms/EnableAdminUpdates.
 
 If AdministratorUpdatesEnabled is disabled but AdministratorUpdatesOptOut is not, AdministratorUpdatesEnabled is respected.</string>
 
+      <string id="AdministratorUpdatesEnabled_Disabled">Administrator Updates Disabled</string>
```
[the remark this answers is not in the document]
Comment (edwardskrod) [reply settingFiles/admx/en-US/VisualStudio.adml:76]: Whoops, I thought I deleted it.

**Comment by edwardskrod** [comment]
> A presentation element represents parameters for a policy setting.  In this case, we are using it to display enum values that we want to display as a drop down for admins to select.

https://learn.microsoft.com/en-us/previous-versions/windows/desktop/Policy/schema-root

**Comment by edwardskrod** [comment]
> I'm struggling a bit to understand the screenshots. I see Enabled with an Options section. Is that a dropdown in the options that allows you to change between our two enabled values?

Yes.  I added an animated gif that should make it clearer!

**Comment by edwardskrod** [comment]
@tydunkel I'm still waiting to hear from @christineruana about the text.  Will wait to push until she approved.

**Review by tydunkel — APPROVED** [review]

**Comment by christineruana** [comment]
My only comment is that we should be using the "MU" acronym instead of "AU".  I think "AU" is more of the term that's internal (like with the publishing tools), but "Microsoft Updates", or "Receive updates from other Microsoft products" or "AllowMUUpdateService" is how it's communicated publicly.

File: settingFiles/admx/VisualStudio.admx [hunk 2]
```diff
@@ -178,16 +178,44 @@
...
+                  </item>
+            </disabledList>
+            <elements>
+                <enum id="AdministratorUpdatesEnabledDropID" valueName="AdministratorUpdatesEnabled">
+                    <item displayName="$(string.AdministratorUpdatesEnabled_EnabledForCatalogAndWSUS)">
+                        <value>
+                            <decimal value="1" />
+                        </value>
+                    </item>
+                    <item displayName="$(string.AdministratorUpdatesEnabled_EnabledForCatalogWSUSAndAU)">
```
Comment (christineruana) [inline settingFiles/admx/VisualStudio.admx:212]: should say MU instead of AU

File: settingFiles/admx/en-US/VisualStudio.adml [hunk 3]
```diff
@@ -73,6 +73,9 @@ For more information, see https://aka.ms/EnableAdminUpdates.
 
 If AdministratorUpdatesEnabled is disabled but AdministratorUpdatesOptOut is not, AdministratorUpdatesEnabled is respected.</string>
 
+      <string id="AdministratorUpdatesEnabled_EnabledForCatalogAndWSUS">Administrator Updates Enabled for Catalog and WSUS</string>
+      <string id="AdministratorUpdatesEnabled_EnabledForCatalogWSUSAndAU">Administrator Updates Enabled for Catalog, WSUS and Automatic Updates</string>
```
Comment (christineruana) [inline settingFiles/admx/en-US/VisualStudio.adml:77]: "and Microsoft Updates"  or  "and updates for other Microsoft products."

File: settingFiles/admx/en-US/VisualStudio.adml [hunk 4]
```diff
@@ -73,6 +73,9 @@ For more information, see https://aka.ms/EnableAdminUpdates.
 
 If AdministratorUpdatesEnabled is disabled but AdministratorUpdatesOptOut is not, AdministratorUpdatesEnabled is respected.</string>
 
+      <string id="AdministratorUpdatesEnabled_EnabledForCatalogAndWSUS">Administrator Updates Enabled for Catalog and WSUS</string>
+      <string id="AdministratorUpdatesEnabled_EnabledForCatalogWSUSAndMU">Administrator Updates Enabled for Catalog, WSUS and Microsoft Updates</string>
```
Comment (edwardskrod) [inline settingFiles/admx/en-US/VisualStudio.adml:77]: @christineruana Look good?

Could you please approve the PR?

**Review by christineruana — APPROVED** [review]

**Comment by artemydmitriev** [comment]
Guys, could someone please exclude the whole company from getting the whole thread of notifications for this PR? Thank you. 😃

**Pull request merged**